views/data_summary.erb

```diff
@@ -0,0 +1,355 @@
+                      </tr>
+                    </table>
+                  </div>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+        <p>
+          <span class="superscript">&dagger;</span>
+          Values of fields marked &dagger; will be consistent throughout the
+          legislature: if their values are equal (string comparison), then
+          they’re the same thing. So all politicians with the same
+          <code>area_id</code>, for example, are representing the same area.
+          Where appropriate, the actual value might be a useful value. For
+          example, where we can, we use <a
+          href="http://opencivicdata.readthedocs.org/en/latest/ocdids.html">Open
+           Civic Data IDs</a> like this:
+          <code>ocd-division/country:us/state:ca/cd:47</code>, but if no such
+          mapping is available from our sources, we’ll probably use a slug like
+          <code>area-foo-county</code>.
+        </p>
+        <h2>
+          Legislatures and terms
+        </h2>
+        <p>
+          For every country, we break the data down into
+          <strong>legislatures</strong> (for example, the “parliament” or
+          “congress” of the country). Currently, we’re only working with
+          national, that is, top-level, legislatures (as opposed to local or
+          state-level legislatures). In the future we hope to include those too.
+        </p>
+        <p>
+          Many countries have a single legislature. Some have two. For example,
+          the <a
+          href="http://everypolitician.org/united-states-of-america/">United
+          States of America</a> has the <em>House of Representatives</em> and
+          the <em>Senate</em>.
+        </p>
+        <p>
+          Furthermore, we split the data for those down into
+          <strong>terms</strong>. Like legislatures themselves, the way this
+          works may vary from country to country. Terms have a start and end
+          date, and the data therefore describes the membership of the
+          legislation during that period. In most democracies, new terms begin
+          after every national election. Of course, changes still occur within
+          terms -- the point is, that’s how we break the data files down for
+          you.
+        </p>
+        <p>
+          This is pragmatic because we know online services and sites are
+          likely to be concerned with the most recent term -- that is, a
+          country’s <em>current</em> legislature.
+        </p>
+        <h2>
+          Politicians
+        </h2>
+        <p>
+          <strong>The richness of the data we have will depend on the source or
+          sources from which we’re getting it.</strong> We try to use official
+          sources, but this isn’t consistently useful across the world. Indeed,
```

Comment:
The "we try to use official sources" part isn't really accurate. As an aspiration it might be, but there are lots of places where we use unofficial sources even if the official source might also be good.